mysql/util.go

```diff
@@ -93,3 +97,11 @@ func GetDefaultFieldLengthAndDecimalForCast(tp byte) (flen int, decimal int) {
 	}
 	return -1, -1
 }
+
+// GetDefaultExplainFormat return the processed values for format of explain
+func GetDefaultExplainFormat(f string) string {
+	if strings.Compare(strings.ToUpper(f), "TRADITIONAL") == 0 {
+		return "row"
+	}
+	return f
```

Comment:
I think this conversion should be done in TiDB.

Reply:
Agree...

How about the `Restore()` of that statement?

Reply:
it has removed.